Sign-up page/readme.md

```diff
@@ -0,0 +1,20 @@
+# Welcome to a basic sign-up page
+
+## Features :
+This sign up page is a very basic one which takes user info like
+1. Name
+2. E-mail Addresss
+3. Contact Number
+4. Password
+
+## Tech Stack Used
+<span>
+  <img src="https://cdn-icons-png.flaticon.com/512/1051/1051277.png" width=40px height=40px> &nbsp; &nbsp;
+  <img src="https://cdn-icons-png.flaticon.com/512/732/732190.png" width=40px height=40px>
+</span>
+
+## Quick View
+<img src="https://github.com/srijanishere/WebDev-ProjectKart/blob/master/Sign-up%20page/Assets/Screenshot%202022-04-03%20at%2010.46.09%20AM.png">
```

Comment:
using some descriptive name for the image would be really nice

Reply:
I've changed the name of the image file with a descriptive name. Hope this will solve the issue.